x/ibc/04-channel/keeper/timeout.go

```diff
@@ -99,13 +109,6 @@ func (k Keeper) TimeoutPacket(
 		return nil, sdkerrors.Wrap(types.ErrInvalidPacket, "packet verification failed")
 	}
 
-	k.deletePacketCommitment(ctx, packet.GetSourcePort(), packet.GetSourceChannel(), packet.GetSequence())
```

Comment:
@cwgoes shouldn't we call `k.TimeoutExecuted` here then? can we add a code comment if not?

Reply:
I guess `TimeoutExecuted` is supposed to be called after the timeout is actually handled. I'm not totally sure.

Reply:
Made an issue - https://github.com/cosmos/cosmos-sdk/issues/5511